### DISCUSSION TITLE
REG- Disable Adding New records in the Child table

### DISCUSSION THREAD

**Discussion by Vinith-kumar47** [serenity-is/Serenity#7049] [Help]
Hi All,
I have Parent and child table in my dialog. And Added one button says "Moved to other table". Once the master and child tables data is  filled. User will summit the form and click the "Moved to other table" button.

once the data is moved to other table, then the child table must not have the option to add new records.Only have the option to update and delete the Records.

How to add the condition to not allow the new records in child table?

Can anyone help me on this. It would be really grateful.

**Comment by LSNicholls** [comment]
Make a public method on your grid if it needs to be called by the parent dialog code, which presumably is where the "Moved to the other table" is happening. As part of that "Moved to the other table" action, call the method of the grid, which will do something like this:

`this.toolbar.findButton("add-button").toggle(false);`

... no?
